libqtile/group.py

```diff
@@ -90,16 +93,33 @@ def current_window(self, win):
             pass
         self.focus_history.append(win)
 
-    def _remove_from_focus_history(self, win):
```

Comment:
I could have added this back in after the most recent changes (and I still can, if it is wanted). But I really don't like this function - it seems overly long, is only used once, and the name doesn't reflect what actually needs to be done that one time.